_posts/2023-03-20-decommissioning-postgresql-database.adoc

```diff
@@ -0,0 +1,162 @@
+ public | preference_preferences                          | table | pgche
+ public | profile                                         | table | pgche
+ public | profile_attributes                              | table | pgche
+ public | projectattribute                                | table | pgche
+ public | projectattribute_values                         | table | pgche
+ public | projectconfig                                   | table | pgche
+ public | projectconfig_mixins                            | table | pgche
+ public | schema_version                                  | table | pgche
+ public | sequence                                        | table | pgche
+ public | serverconf                                      | table | pgche
+ public | serverconf_attributes                           | table | pgche
+ public | serverconf_properties                           | table | pgche
+ public | sourcestorage                                   | table | pgche
+ public | sourcestorage_parameters                        | table | pgche
+ public | sshkeypair                                      | table | pgche
+ public | user_aliases                                    | table | pgche
+ public | userdevfile                                     | table | pgche
+ public | usr                                             | table | pgche
+ public | workspace                                       | table | pgche
+ public | workspace_attributes                            | table | pgche
+ public | workspaceconfig                                 | table | pgche
+(84 rows)
+----
+
+However, with the switch to link:https://che.eclipseprojects.io/2021/10/12/@mario.loriedo-devfile-v2-and-the-devworkspace-operator-p1.html[DevWorkspace Operator and Devfile v2], all this data becomes irrelevant since we started using native Kubernetes resources instead.
+
+The data is now stored and managed as Kubernetes resources such as Secrets and ConfigMaps in the user namespaces:
+
+image::/assets/img/decommissioning-postgresql-database/secrets.png[Secrets from the user namespace]
+
+image::/assets/img/decommissioning-postgresql-database/configmaps.png[ConfigMaps from the user namespace]
+{nbsp} +
+
+As an example, SSH keys are now configured using the dedicated link:https://github.com/devfile/devworkspace-operator/blob/main/docs/additional-configuration.adoc#configuring-devworkspaces-to-use-ssh-keys-for-git-operations[secret].
+
+Starting from the {prod} `7.62.0` release the PostgreSQL deployment will cease to exist, resulting in only 5 operands: Che Server, User Dashboard, Plugin Registry, Devfile Registry, and Gateway.
+
+image::/assets/img/decommissioning-postgresql-database/operands.png[{prod} operands]
+{nbsp} +
+
+IMPORTANT: No manual migration steps and action items are required. The PostgreSQL deployment will be gracefully terminated and removed during the update. All the obsolete data stored in the database will be removed.
```

Comment:
```suggestion
IMPORTANT: No manual migration steps or action items are required for this change to take effect. The PostgreSQL deployment will be gracefully terminated and removed during the update. All the obsolete data stored in the database will be removed.
```

Reply:
Not sure how to phrase it, but it'd be good to add something like: '...and removed during the update to any version >= 7.62'